dotcom-rendering/src/web/components/FixedMediumFastXI.tsx

```diff
@@ -43,15 +46,40 @@ export const FixedMediumFastXI = ({
 	containerPalette,
 	showAge,
 }: Props) => {
-	const firstSlice = trails.slice(0, 3);
+	const firstSlice50 = trails.slice(0, 1);
+	const firstSlice25 = trails.slice(1, 3);
 	const remaining = trails.slice(3, 11);
 	return (
 		<>
-			<Card50_Card25_Card25
-				cards={firstSlice}
-				containerPalette={containerPalette}
-				showAge={showAge}
-			/>
```

Comment:
Was this slice no longer behaving as required by the layout?

Reply:
It was behaving differently only for the 0 sublinks case as it wasn't showing the trailtext so I thought I'd change it to use the new wrappers.

![Screenshot 2023-01-11 at 16 33 14](https://user-images.githubusercontent.com/19683595/211863637-f0264809-e3ef-44d0-ab8c-6b206cdd6159.png)

Reply:
Ah okay! Only thing I wasn't sure on is whether that was the correct behaviour for that card wrapper (or should the card wrapper be receiving the sublink support here)

Reply:
If I understand the question correctly, if we wanted we could change `Card25Media25` to `Card25Media25Tall`[ in the `Card50_Card25_Card25 ` wrapper](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/dynamicSlices.tsx#L240) instead of here. 

One thing I just noticed is that `Card50_Card25_Card25` is only used in `FixedMediumFastXI` so it seems we have two options:
1. Leave the change as it is in this PR + delete `Card50_Card25_Card25`
2. Keep `Card50_Card25_Card25` and add the sublink support there

I'm leaning towards option 1 because I think that's more consistent with the rest of the fixed containers but I don't have a strong preference. Let me know what you think!

Reply:
I had another look and updating my comment above. There were two `Card50_Card25_Card25` wrappers, hence the confusion:
1. [One in `DynamicSlowMPU`](https://github.com/guardian/dotcom-rendering/blob/b88596cf158bb7846bf691f2a20c8cd7450e8e12/dotcom-rendering/src/web/components/DynamicSlowMPU.tsx#L127) which was using `Card25Media25Tall` for the 25% cards
2. [One in `dynamicSlices`](https://github.com/guardian/dotcom-rendering/blob/d67a830f453e9db7962068e3066c891a10f1e547/dotcom-rendering/src/web/lib/dynamicSlices.tsx#L208) which was using the `Card25Media25`.

I kept the one in `dynamicSlices` and changed its 25% cards to use `Card25Media25`.